### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by rishichawda**

### CODE REVIEW

**Comment by Arjun-sna** [comment]
@rishichawda Some CSS files are still inside assets folder. please move them to the appropriate folders.

**Comment by Arjun-sna** [comment]
@rishichawda referring to fine `Product.js`, in function `renderProductActions` we can move the `<button />`  used to components folder. And these are things which will go into components. So please check for such small components and do the same.

**Comment by Arjun-sna** [comment]
@rishichawda also write proper documentation for the components created and add proptyes and defaut prop details.

File: src/templates/product/ProductVariants.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react/index.js'
 import VariantItem from "./VariantItem.js";
```
Comment (Arjun-sna) [inline src/templates/product/ProductVariants.js]: all absolute imports should come before relative imports.

and use single quotes consistently across the project.

Comment (Arjun-sna) [inline src/templates/product/ProductVariants.js]: all absolute imports should come before relative imports.

and use single quotes consistently across the project.

File: src/templates/category/ProductCard.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class ProductCard extends Component {
       <div className="col-lg-4 col-md-6 col-sm-6 col-xs-12">
         <div className="demo-product-card">
           <Link to={`/product/${this.props.productId}`} style={{margin: '0'}}>
-            { this.props.children ||  <img src={this.props.images[0] ? this.props.images[0].originalSrc : require('../assets/images/default.jpeg')} alt={this.props.productName} />}
+            { this.props.children ||  <img src={this.props.images[0] ? this.props.images[0].originalSrc : require('../../assets/images/default.jpeg')} alt={this.props.productName} />}
```
Comment (Arjun-sna) [inline src/templates/category/ProductCard.js]: `<img src={this.props.images[0] ? this.props.images[0].originalSrc : require('../../assets/images/default.jpeg')` add a method and move this logic there and finally return only what to render.

File: src/layouts/index.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import Helmet from 'react-helmet';
-import './index.scss'
-import '../components/css/custom.scss';
+import '../assets/styles/index.scss'
```
Comment (Arjun-sna) [inline src/layouts/index.js]: we can follow below import order
1. Api Dependencies
2. Component Dependencies
3. CSS, assets

**Comment by rishichawda** [comment]
Changes pushed as mentioned in the comments. Please check and let me know if any other changes are required.

**Pull request merged**